internal/proxier/proxy.go

```diff
@@ -221,9 +224,26 @@ func (p *Proxier) reconcile(key string) error {
 		}
 		return nil
 	}
+
 	svc := o.(*corev1.Service)
+	if svc.Spec.Type != corev1.ServiceTypeClusterIP {
+		// Don't process non-clusterip services.
+		return nil
+	}
+
+	var skip bool
```

Comment:
This is inefficent, but I figured this shouldn't be in a hot enough code path that this would matter.